multiqc/utils/search_patterns.yaml

```diff
@@ -60,6 +60,8 @@ htseq:
     contents: '__too_low_aQual'
 hicup:
     fn: 'HiCUP_summary_report*'
+jellyfish:
+    fn: '*.hist'
```

Comment:
Well also the `.jf` extension is only the default, a user can override it so anyway an explanation is needed.
I will add a paragraph in the docs, and search for files with extension `*_jf.hist`.... a possible feature is that a user can specify a pattern to be searched, something like `--jellyfish-extension 'fn:*.hist;contents:\'content to be searched\''`